frontend/src/index.html

```diff
@@ -33,13 +33,24 @@
     <!-- 게임 캔버스 -->
     <div id="gameCanvasContainer" class="position-relative h-100 w-100 d-none">
       <canvas id="gameCanvas" tabindex="0" class="h-100 w-100"></canvas>
-      <div class="position-absolute top-0 px-3 text-white d-flex flex-column align-items-center border border-5 rounded">
+      <div id="scoreContainer" class="position-absolute top-0 w-40 px-3 text-white d-flex flex-column align-items-center border border-5 rounded">
```

Comment:
```suggestion
      <div id="scoreContainer" class="position-absolute top-0 start-50 translate-middle-x w-40 px-3 text-white d-flex flex-column align-items-center border border-5 rounded">
```
화면 줄였을 때도 위치 항상 가운데 오도록 start, translate 추가해주면 좋을  것 같습니다!